t/mojo/streams.t

```diff
@@ -0,0 +1,175 @@
+use Mojo::Base -strict;
+
+BEGIN { $ENV{MOJO_REQUEST_TIMEOUT} = 0.25 }
```

Comment:
What is this used for?

Reply:
There should be a `BEGIN { $ENV{MOJO_REACTOR} = 'Mojo::Reactor::Poll' }` here, to avoid accidental uses of `EV`.